### ISSUE TITLE
GitHub URL being added to "Update This Repo Only" in 1.5.6

### ISSUE THREAD

**Issue by Lotusshaney** [lindegroup/autopkgr#666]
Trying to update a single repo by right clicking and selecting "Update This Repo Only" results in https://github.com/ being added to the local URL and errors out.

See Attached picture
![Screenshot 2021-03-02 at 15 29 54](https://user-images.githubusercontent.com/1845849/109676702-e1378d80-7b70-11eb-9a5b-50e3b85c4ed5.png)

This happens with all repos

**Comment by jazzace** [comment]
Can confirm. I selected 2 repos that needed updating (🟡) and used the contextual menu to "Update Selected Repos". I received an error message for both of the kind shown in the screenshot above. I then went to Terminal and tried to update them from there:
```
% autopkg repo-update jazzace-recipes homebysix-recipes
ERROR: Can't find an installed repo for https://github.com/autopkg/jazzace-recipes
Attempting git pull for /Users/tech/Library/AutoPkg/RecipeRepos/com.github.autopkg.homebysix-recipes...
Updating 451b3e7..ae5053b
Fast-forward
 Zscaler/Zscaler.munki.recipe | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```
The interesting thing here is that the second one, which threw an error previously, was correctly updated.
So I ran the `repo-update` command again for the problem repo:
```
% autopkg repo-update jazzace-recipes                  
ERROR: Can't find an installed repo for https://github.com/autopkg/jazzace-recipes
```
So I then tried to delete and re-add:
```
% autopkg repo-delete jazzace-recipes && autopkg repo-add jazzace-recipes
ERROR: Can't find an installed repo for https://github.com/autopkg/jazzace-recipes
Attempting git pull for /Users/tech/Library/AutoPkg/RecipeRepos/com.github.autopkg.jazzace-recipes...
Updating e975fda..c8d8a0f
Fast-forward
 Zoom/Zoom-ForIT.pkg.recipe | 83 ++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 83 insertions(+)
 create mode 100644 Zoom/Zoom-ForIT.pkg.recipe

Updated search path:
[…]
 ```
It was unsuccessful in deleting it, but the `repo-add` verb still saw the local copy and simply updated the repo. AutoPkgr then correctly confirms that the local copy of the repos are up to date.

I did a further test on a repo that did not need updating (and for some reason did not have a 🟢 or any status indicator at all, but that's a separate bug report) and it threw the same error, but checking it at the command line with `repo-update` reported `Already up-to-date`, so it does not seem to do damage in that instance.

**Comment by apizz** [comment]
+1

**Comment by shawnhonsberger** [comment]
I'm unable to reproduce this. Are you running AutoPkg 2.3?

**Comment by apizz** [comment]
Yes for me

**Comment by jazzace** [comment]
I am using AutoPkg 2.3 as well. I had an un-updated copy on another machine, so here's what I found:
2.2 + 1.5.5 = success
2.3 + 1.5.5 = fail
And my previous report was:
2.3 + 1.5.6 = fail

So it looks like the issue is caused by a change in AutoPkg 2.3, not a change in AutoPkgr 1.5.6.

**Comment by shawnhonsberger** [comment]
Interesting, thanks for the follow up @apizz and @jazzace.

**Comment by Lotusshaney** [comment]
Have checked with AutoPkg 2.3.1 and this appears to fix this problem

**Comment by apizz** [comment]
Confirmed this is resolved with autopkg 2.3.1

**Comment by shawnhonsberger** [comment]
Thanks @Lotusshaney!
